lib/tasks/backfill_type_for_job_applications.rake

```diff
@@ -0,0 +1,6 @@
+namespace :job_applications do
+  desc "Backfill job application type where it's nil"
+  task backfill_type: :environment do
+    JobApplication.where(type: nil).update_all(type: "NativeJobApplication")
```

Comment:
Vacancy defaulted its type column which feels slightly cleaner than needing to run this script